<?xml version="1.0" encoding="UTF-8"?>
<!-- 
    This is the production launch file to be used with the real roboclaw device.
-->
 <launch>

    <node name="roboclaw" pkg="roboclaw_driver" type="roboclaw_node.py">
        <param name="test_mode" type="bool" value="false"/>
        <param name="dev_name" type="str" value="/dev/ttyACM0"/>
        <param name="baud" type="int" value="115200"/>
        <param name="address" type="int" value="128"/>
        <param name="loop_hz" type="int" value="100"/>
        <param name="deadman_secs" type="int" value="3"/>
        <param name="speed_cmd_topic" type="str" value="roboclaw/speed_command"/>
        <param name="stats_topic" type="str" value="roboclaw/stats"/>        

        <param name="max_abs_linear_speed" value="1.5"/>
        <param name="max_abs_angular_speed" value="3"/>
        <!-- 1957 ticks/rev ; 0.2 cm diam ; 0.6283186 m/rev -->
        <param name="ticks_per_meter" value="1900"/>
        <param name="base_width" value="0.4"/>
    </node>

</launch>